### PR TITLE
Component Unique ID Bugfix

### PR DESCRIPTION
**Pull request by nirvanpagooah**
It happens that the Date.now() method does not produce a unique ID and Vue renders within nano-seconds causing all script tags to aggregate together.

I have applied a fix which generates a unique ID with a random string.

Hope that help!

### CODE REVIEW

File: src/components/VueScriptComponent.vue [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
+      .toString(36)
+      .substring(7);
     return {
-      compId: Date.now() //we will use date.now as a soft ID so that users may load multiple vue-script-components without conflicting ids
+      compId: window.performance.now() + "-" + randomID
     };
   },
-
+  props: {
+    id: {
```
Comment (TheDynomike) [inline src/components/VueScriptComponent.vue]: this prop is always null?

File: src/components/VueScriptComponent.vue [hunk 2]
```diff
@@ -1,44 +1,48 @@
 <template>
-  <div :id="compId">
-  </div>
+  <div :id="compId" :key="id"></div>
```
Comment (TheDynomike) [inline src/components/VueScriptComponent.vue]: what is the purpose of the :key here?

File: src/components/VueScriptComponent.vue (hunk 1 above)
Comment (nirvanpagooah) [reply src/components/VueScriptComponent.vue]: The `id` prop has been removed in the latest commit.

File: src/components/VueScriptComponent.vue (hunk 2 above)
Comment (nirvanpagooah) [reply src/components/VueScriptComponent.vue]: My apologies. It was a duplicate on my end.

**Comment by nirvanpagooah** [comment]
I am now using `window.performance.now()` to generate a more unique random value. Date.now() happens to be same on rendering.

Kindly test and update. I am already using the forked version on a production site for Ads.

**Pull request merged**